<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EndpointInWidget</class>
 <widget class="QWidget" name="EndpointInWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>261</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
   <item>
    <layout class="QVBoxLayout" name="layoutButtons">
     <item>
      <widget class="QLabel" name="labelEndpointName">
       <property name="text">
        <string>EP UNKOWN</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSetBufferSize">
       <property name="text">
        <string>Set Buffer Size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonReadOnce">
       <property name="text">
        <string>Read Once</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonKeepRead">
       <property name="text">
        <string>Keep Read</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="hexEditPlaceHolder" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
